tokio/src/task/spawn.rs

```diff
@@ -121,6 +121,24 @@ cfg_rt! {
     /// ```text
     /// error[E0391]: cycle detected when processing `main`
     /// ```
+    /// =================================================
+    /// 
+    ///Warning: Large futures may cause a stack overflow! 
+    ///
+    ///Example:
+    /// ```no_run
+    ///#[tokio::main]
+    ///async fn main() {
+    ///     tokio::spawn(async move {
+    ///         let mut buf = [0; 2097151];
+    ///     }).await;
+    ///}
+    ///```
+    ///thread 'tokio-runtime-worker' has overflowed its stack
+    /// 
+    ///fatal runtime error: stack overflow 
+    /// 
+    ///timeout: the monitored command dumped core
```

Comment:
All doc-comments should start with three slashes followed by a space. You need to introduce an extra space where its missing. Additionally, please don't include error messages in raw text - they need to be in code blocks too. However here I'd prefer to just remove the error message.